manifest.yaml

```diff
@@ -10,4 +10,5 @@ applications:
       - python_buildpack
     command: gunicorn s3.main:app
     services:
-      - sk-s3
+      - ((VENDOR_1))
```

Comment:
instead of calling them vendors, I would refer to them as services or buckets

Reply:
Renamed to services